tests/ControllerTest.php

```diff
@@ -60,7 +60,7 @@ public function testFilesystemAndDatabase(): void
     public function testExcludeShouldBePresentInRsyncCommand(): void
     {
         $config = new GlobalConfig();
-        $config->addFilesystem('fileadmin', 'fileadmin')->addExclude('*.mp4')->addExclude('*.mp3')->addExclude('*.zip');
+        $config->addFilesystem('fileadmin', 'fileadmin')->addExclude('*.mp4')->addExclude('*.mp3')->addExcludes(['*.zip', '*.rar']);
```

Comment:
add Exclude should be a test on its own - a test should only test thing at a time...
what do you think @Kanti ?

Reply:
I don't have a strong opinion on this.

Reply:
I do not know it either. If you like you can add a test and just push the commit into the branch.